### DISCUSSION TITLE
[feature request]: Autofill project (custom) fields with data from issue form

### DISCUSSION THREAD

**Discussion by ghost** [community/community#58131] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

Issues and projects are getting integrated more and more. For example, there's a workflow so that when an issue is raised in a repository, it automatically lands in a specific project, which is great.

However, very often we need specific information from that issue to populate the project. More specifically, we have created [custom issue forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) with fields that are needed in our project. Unfortunately, although the issue lands automatically in the project board, we still need to fill the project's (custom) fields by hand from the issue. It would be amazing if this could be done automatically, since it would save our team tons of time.

This is similar to what is mentioned in this [discussion](https://github.com/orgs/community/discussions/45641), but expands to go beyond just a date field.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by 65156** [comment]
agree, cant believe this isnt already a thing!

**Comment by aikenka** [comment]
Upvoted - right now there's only one workflow allow to set one field value when an item is added to a project.

It would be nice if the auto-add to project workflow allowed to set field values as well.

**Comment by mcw0933** [comment]
Chiming in to say the same!  Very wanted capability
